cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -64,6 +65,60 @@ func getInternalTypeFor(context *generator.Context, t *types.Type) (*types.Type,
 	return context.Universe.Package(internalPackage).Type(t.Name.Name), true
 }
 
+type conversionType struct {
+	inType  *types.Type
+	outType *types.Type
+}
+type conversions map[conversionType]*types.Type
```

Comment:
Actually all of them are DeclarationOf and the underlying type is Func. Adding a comment.